
Taken from the Linux Tegra tree.  Patch has been accepted for 4.8-rc1

Origin: https://git.kernel.org/cgit/linux/kernel/git/tegra/linux.git/log/?h=for-4.8/arm64/dt


commit 9168e1db754383ea16b75ffcc1efbb9b09cfae64
Author: Jon Hunter <jonathanh@nvidia.com>
Date:   Wed Jun 29 12:07:33 2016 +0100

    arm64: tegra: Correct Tegra210 XUSB mailbox interrupt
    
    The XUSB mailbox interrupt for Tegra210 is 40 and not 49 which is for
    the XUSB pad controller. For some Tegra210 boards, this is causing USB
    connect and disconnect events to go undetected. Fix this by changing the
    interrupt number for the XUSB mailbox to 40.
    
    Signed-off-by: Jon Hunter <jonathanh@nvidia.com>
    Signed-off-by: Thierry Reding <treding@nvidia.com>

diff --git a/arch/arm64/boot/dts/nvidia/tegra210.dtsi b/arch/arm64/boot/dts/nvidia/tegra210.dtsi
index ac1fb88..b731880 100644
--- a/arch/arm64/boot/dts/nvidia/tegra210.dtsi
+++ b/arch/arm64/boot/dts/nvidia/tegra210.dtsi
@@ -629,7 +629,7 @@
 		reg-names = "hcd", "fpci", "ipfs";
 
 		interrupts = <GIC_SPI 39 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH>;
+			     <GIC_SPI 40 IRQ_TYPE_LEVEL_HIGH>;
 
 		clocks = <&tegra_car TEGRA210_CLK_XUSB_HOST>,
 			 <&tegra_car TEGRA210_CLK_XUSB_HOST_SRC>,
